src/librustc_mir/dataflow/impls/mod.rs

```diff
@@ -542,7 +542,7 @@ impl<'a, 'gcx, 'tcx> BitDenotation for MovingOutStatements<'a, 'gcx, 'tcx> {
     }
 
     fn propagate_call_return(&self,
-                             in_out: &mut IdxSet<MoveOutIndex>,
+                             sets: &mut BlockSets<'_, MoveOutIndex>,
```

Comment:
This comment may be made irrelevant by later commits in the series, but I don't think it was correct to replace `IdxSet` with `BlockSets` here.

IIRC, `BlockSets` is a triple of sets associated with the block currently being processed. But `fn propagate_call_return` needs to modify the successor block for the call terminator, which is why we pass in the whole `in_out` here and not just the three sets for the current block.

Reply:
(I think a commit further down the line did indeed make this feedback irrelevant. If it was ever relevant in the first place, which I am no longer certain of; the `IdxSet` here is not for the whole universe of blocks anyway...)